src/main/java/bio/terra/cli/command/cromwell/GenerateConfig.java

```diff
@@ -22,6 +30,21 @@ public class GenerateConfig extends BaseCommand {
 
   @Override
   protected void execute() {
+    // Get the absolute path for the generate-cromwell-config.sh file.
+    String absolutePath = "";
+    try {
+      File file = File.createTempFile("generate-cromwell-config", ".sh");
+      Path path = file.toPath();
+      InputStream inputStream =
+          getClass().getClassLoader().getResourceAsStream("configs/generate-cromwell-config.sh");
+      // Copy the content in script and grant read, write, execute permission.
+      Files.copy(inputStream, path, StandardCopyOption.REPLACE_EXISTING);
+      Files.setPosixFilePermissions(path, PosixFilePermissions.fromString("rwxrwxrwx"));
+      absolutePath = path.toString();
+    } catch (IOException e) {
+      OUT.println("Failed to write to cromwell config file");
```

Comment:
return here since the rest can't be executed.

then you don't need to check for the aboslutepath is empty case below.